### DISCUSSION TITLE
Cache all artwork setting

### DISCUSSION THREAD

**Discussion by djnield** [croneter/PlexKodiConnect#1394] [Q&A] [answered]
I've had to disable the cache all artwork for a smooth kodi experience. Library is too big and it totally ate through all the space in my shield. 11Gb used vs 4Gb.

But what's the difference of having this enabled vs disabled as I still see artwork obviously?

**Comment by croneter** [comment, marked as answer]
If enabled, pkc cashes all artwork proactively. Browsing through your library seems smoother as pictures do not need to be loaded from the Plex Media Server first. If disabled, you will notice that artwork is missing when you scroll through your Kodi library really fast - it takes a bit longer to load
